webui/react/src/pages/FlatRuns/FlatRuns.tsx

```diff
@@ -986,6 +1015,11 @@ const FlatRuns: React.FC<Props> = ({ projectId, searchId }) => {
               rowHeight={globalSettings.rowHeight}
               onRowHeightChange={onRowHeightChange}
             />
+            {!isMobile && (
+              <span className={css.runSelection} data-test="runSelection">
```

Comment:
can we add test cases?

Reply:
i think tests need to be in the same PR. AFAIK, that's what we agreed in the meeting for the reliability